primary/tests/causal_completion_tests.rs

```diff
@@ -109,35 +104,31 @@ async fn test_read_causal_signed_certificates() {
 
     // Enabled debug tracing so we can easily observe the
     // nodes logs.
-    setup_tracing();
+    let _guard = setup_tracing();
```

Comment:
We need to hold the guard here other wise the subscription will be dropped and logs are captured